samples/AzCopyCore/AzCopyCore/AzCopyCore.csproj

```diff
@@ -19,18 +21,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26223-02" />
-    <PackageReference Include="System.Memory" Version="$(CoreFxPackageVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(CoreFxPackageVersion)" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26320-02" />
+    <PackageReference Include="System.Buffers.ReaderWriter" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Memory" Version="4.5.0-preview3-26319-04" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.0-preview3-26319-04" />
```

Comment:
Update and use the CoreFxPackageVersion property to make it easier to update in the future.

Reply:
As I said above, I like how the sample shows VS experience that our customers will get.

Reply:
I don't know if someone would go and manually update each individual referenced packages using VS/NuGet package manager. Once they are added, it would make sense to create a property to update the versions, no?